packages/ketcher-polymer-editor-react/src/components/Layout/Layout.tsx

```diff
@@ -0,0 +1,112 @@
+/****************************************************************************
+ * Copyright 2021 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ***************************************************************************/
+
+import React from 'react'
+import styled from '@emotion/styled'
+
+interface LayoutProps {
+  children: JSX.Element | Array<JSX.Element>
+}
+
+const PADDING = '15px'
+
+const LayoutContainer = styled.div(({ theme }) => ({
+  display: 'flex',
+  justifyContent: 'space-between',
+  height: '100%',
+  width: '100%',
+  position: 'relative',
+  padding: `${PADDING}`,
+  backgroundColor: theme.color.background.canvas
+}))
+
+const CenterContainer = styled.div({
+  display: 'flex',
+  flexDirection: 'column',
+  width: '100%',
+  height: `calc(100vh - 40px - ${PADDING} * 2)`
+})
+
+const Left = styled.div({
+  width: '32px',
+  minWidth: '32px',
+  height: `calc(100vh - 40px - ${PADDING} * 2)`
+})
```

Comment:
Note that this notation allows passing of all the props of a div element, so user is able to do this:
```tsx
...
<Layout.Left style={{position: "absolute", right: 0}}>
  <p>Why am I on the right?</p>
</Layout.Left>
...
```
We would still need to somehow enclose it to a React component to control which props can be passed and to avoid breaking layout.

Reply:
I found that we can do it like this ```
styled('div', {
  shouldForwardProp: (prop) => prop === 'children'
})
```